deploy/docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,61 @@
+version: "3.3"
+
+services:
+  vsc-node: # name of the service
+    image: platinium/vsc-node:latest
```

Comment:
@vaultec81 image name needs to be adjusted in respect to whats choosen for DOCKERHUB_IMAGE

Reply:
Going to use my personal account for now